data_source_obmcs_core_instance.go

```diff
@@ -161,3 +168,18 @@ func (s *InstanceDatasourceCrud) SetData() {
 	}
 	return
 }
+
```

Comment:
Please test if this is still necessary and if so add a brief note explaining the problem and how this solves it.